timetracker/timetracker/views.py

```diff
@@ -0,0 +1,12 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+
+def name_form(request):
+	return render(request, 'name-form.html')
+
+def hello(request):
+	if 'name' in request.GET:
+		received = request.GET.get("name", "World")
+		if len(received) == 0:
```

Comment:
Note that an empty string is considered falsey, so this could have been `if not receved:`
Further, the default value for `request.GET.get` is None, so you could used:

```
received = request.GET.get('name')
if not received:
    received = 'World'
```